checkout-post-purchase/src/index.liquid

```diff
@@ -133,7 +133,7 @@ import {
  * optionally allows data to be stored on the client for use in the `Render`
  * extension point.
  */
- extend("Checkout::PostPurchase::ShouldRender", async ({ storage }) => {
+extend("Checkout::PostPurchase::ShouldRender", async ({ storage }) => {
```

Comment:
Let's revert this change to remove the space

Reply:
@vividviolet  I saw this as well, but I think the space was off before. It wasn't aligned with the functions around it.

Reply:
Oops, you're right. Ignore me